main.js

```diff
@@ -160,4 +160,174 @@ const inventory = [
     originalStock: 10,
     sold: 8,
   },
-];
\ No newline at end of file
+];
+
+// Opdracht 1a
+let tvNames = []
+inventory.map(item => {
+    tvNames.push(item.name)
+})
+
+console.log(tvNames)
+
+// Opdracht 1b
+let soldOut = []
+
+inventory.filter(tv => {
+    if (tv.originalStock - tv.sold <= 0) {
+        soldOut.push(tv)
+    }
+})
```

Comment:
Uiteindelijk geeft de filter functie een array terug:
```suggestion
const soldOut = inventory.filter(tv => {
    return tv.originalStock === tv.sold;
})
```